app/helpers/cart_helper.rb

```diff
@@ -0,0 +1,16 @@
+module CartHelper
+  def get_name_product pro_id
+    product = Product.find_by(id: pro_id)
+    product.name
```

Comment:
Dùng toán tử an toàn đi em. Chứ find by mà nil sửa lại .name thì tèo

Reply:
dạ chị